acoular/fastFuncs.py

```diff
@@ -56,9 +55,10 @@ def calcCSM(csm, SpecAllMics):
 
 
 def beamformerFreq(steerVecType, boolRemovedDiagOfCSM, normFactor, inputTupleSteer, inputTupleCsm):
-    r"""Conventional beamformer in frequency domain. Use either a predefined
-    steering vector formulation (see Sarradj 2012) or pass your own
-    steering vector.
+    r"""Conventional beamformer in frequency domain.
```

Comment:
```suggestion
    r"""
    Conventional beamformer in frequency domain.
```

Again `GL01 Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)`